server/bin/pbench-index.py

```diff
@@ -36,6 +37,28 @@
 from pbench.server.utils import rename_tb_link, quarantine
 
 
+class SigIntException(Exception):
+    pass
+
+
+class SigTermException(Exception):
+    pass
+
+
+def sigint_handler(signum, frame):
+    raise SigIntException
+
+
+def sigterm_handler(signum, frame):
+    raise SigTermException()
```

Comment:
Parens or no parens? Make up your mind :-)

Reply:
Ha; I missed that. Yeah, I assume we probably want to actually construct exception objects in both case.

Reply:
parens